src/utils/legacyLayerParamUtils.js

```diff
@@ -15,22 +16,33 @@ function readUrlParamValue(url, paramName) {
     return undefined
 }
 
-const newLayerParamRegex = /^[\w.]+[@\w=]*[,ft]*[,?\d.]*$/
-
-function isExternalLayer(layerId) {
-    return (
-        layerId &&
-        (layerId.startsWith('WMS|') || layerId.startsWith('WMTS|')) &&
-        layerId.indexOf('||') === -1
-    )
+/**
+ * @param {String} search The query made to the mapviewer
+ * @returns True if the query starts with ? or /?
+ */
+export const isLegacyParams = (search) => {
+    const parts = search?.match(/^(\?|\/\?).*$/g) ?? null
+    return parts && Array.isArray(parts)
```

Comment:
The type check is not needed here you could simply return `return !!search?.match(/^(\?|\/\?).*$/g) ?? false`